skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/LegacyBrevService.kt

```diff
@@ -54,6 +55,10 @@ class LegacyBrevService(
         if (brevtittel.isNullOrBlank()) {
             return BestillOgRedigerBrevResponse(EXSTREAM_BESTILLING_MANGLER_OBLIGATORISK_INPUT)
         }
+        val navansatt = navansattService.hentNavansatt(call , call.principal().navIdent)
+        if(navansatt.resultOrNull()?.navn == null) {
```

Comment:
Vær klar over at ved å bruke `resultOrNull` her så sluker du f.eks internal server error fra NavAnsatt rest kall.
Men om det er greit så ville jeg kjørt `resultOrNull()` på linjen over slik at variabelen har typen `Navansatt`, på den måten så skal det ikke være nødvendig med en non-null-assertion `!!` slik du gjør lenger nede.

Reply:
Ja jeg tenkte på det men det ble mye greier for å håndtere noe som sannsynligvis alltid fungerer, jeg regner med at det er lite logikk bak å hente dette